src/cider/function/CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ add_custom_command(
     ${CMAKE_CURRENT_SOURCE_DIR}/../exec/nextgen/context/ContextRuntimeFunctions.h
     ${CMAKE_CURRENT_SOURCE_DIR}/../exec/nextgen/operators/OperatorRuntimeFunctions.h
     ${CMAKE_CURRENT_SOURCE_DIR}/../exec/nextgen/function/CiderStringFunctions.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/../exec/nextgen/function/CiderSetFunctions.cpp
   OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/RuntimeFunctions.bc
   COMMAND
```

Comment:
can traverse and add all files in function folder automatically?

Reply:
Current the function files exist in different folders and not quite good to use `foreach` to apply all files. We can consolidate these files after fully migrated to nextgen.